### PR TITLE
add unit of account backend components

### PR DESCRIPTION
**Pull request by yushih**
Add three backend components required to provide the unit of account functionality:

1. the service endpoints handlers for coin price queries, using data in DB
2. a crawler (https://github.com/Emurgo/yoroi-graphql-migration-backend/tree/yushi/unit-of-account-backend/script/coin-price-data-fetcher) that fetches the coin price data from external APIs and store the result in S3
3. a poller (https://github.com/Emurgo/yoroi-graphql-migration-backend/blob/yushi/unit-of-account-backend/src/coin-price/poller.ts) that pulls the data in S3 into the DB.

### CODE REVIEW

**Review by vsubhuman — COMMENTED** [review]
/check

File: script/coin-price-data-fetcher/config/default.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+module.exports  = {
+  sourceCurrencies: ['ADA', 'ERG'],
+  targetFiatCurrencies: ['USD', 'JPY', 'EUR', 'CNY', 'KRW', 'BRL'],
+  targetCryptoCurrencies: ['ETH', 'BTC'],
```
Comment (vsubhuman) [inline script/coin-price-data-fetcher/config/default.js:4]: We need to make this into env vars later

Comment (yushih) [reply script/coin-price-data-fetcher/config/default.js:4]: OK. But please know that the reason that I hard-code them here is that they are not simply configurable parameters: to add a new supported currency, `apI.js` is likely to need changes.

File: script/coin-price-data-fetcher/config/default.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+module.exports  = {
+  sourceCurrencies: ['ADA', 'ERG'],
+  targetFiatCurrencies: ['USD', 'JPY', 'EUR', 'CNY', 'KRW', 'BRL'],
```
Comment (stackchain) [inline script/coin-price-data-fetcher/config/default.js:3]: Can we add CAD and IDR? We have a considerable amount of users on mobile.

Comment (vsubhuman) [reply script/coin-price-data-fetcher/config/default.js:3]: We'll add it into backlog

**Review by vsubhuman — APPROVED** [review]

**Pull request merged**